autoinit/estimators/dense.py

```diff
@@ -87,36 +120,46 @@ def estimate(self, means_in: List, vars_in: List):
         """
 
         second_moment = vars_in[0] + math.pow(means_in[0], 2)
-        scale = 1.0 / second_moment
+        scale = self.signal_variance / second_moment
         gain = math.sqrt(scale)
-        distribution = self.estimator_config.get("distribution",
-                                                 "truncated_normal")
+        distribution = self.estimator_config.get("distribution", "truncated_normal")
+        if isinstance(self.layer, DEPTHWISE_CONV):
+            # For depthwise convolutional layers, each channel is convolved with a different
+            # kernel, so we need to multiply the scale/gain by the number of channels.
```

Comment:
Took a bit to figure out this comment, maybe more clear:

"For depthwise convolutional layers, each kernel takes a single channel as input (instead of all channels), so we need to multiply the scale/gain by the number of channels to recover the intended output variance."

Reply:
Yes, much better, thanks!